### PR TITLE
FAQ Update #international_phone_numbers: Update TAN hotline time #2294

### PR DESCRIPTION
**Pull request by browkin**
- Fixes issue #2294 
- changed the times for the contact numbers in the faq

### CODE REVIEW

**Comment by dsarkar** [comment]
closes https://github.com/corona-warn-app/cwa-website/issues/2294

**Comment by MikeMcC399** [comment]
https://github.com/corona-warn-app/cwa-website/issues/2294#issuecomment-1007542853 contains screenshots from 2.16.0-RC6 for comparison.

- https://www.coronawarn.app/en/faq/#who_can_help and
- https://www.coronawarn.app/de/faq/#who_can_help

should also be changed.

File: src/data/faq.json [hunk 1]
```diff
@@ -37,11 +37,11 @@
...
+                            "The technical hotline will help you with technical questions about the Corona-Warn-App, for instance if you have problems with the exposure logging. It is available Monday to Saturday from 7 a.m. to 8 p.m. German local time (except on national holidays)",
                             "<b>National:</b> 0800 7540001",
                             "<b>International:</b> +49 30 498 75401",
                             "<b>TAN Hotline</b>",
-                            "The TAN hotline will help you to enter the TAN if you have a positive PCR test result and if you have not received a TAN or document with a PCR-Test QR code. The TAN hotline is available 24 hours a day, seven days a week.",
+                            "The TAN hotline will help you to enter the TAN if you have a positive PCR test result and if you have not received a TAN or document with a PCR-Test QR code. The TAN hotline is available Monday to Sunday from 7 a.m. to 8 p.m. German local time (except on national holidays)",
```
Comment (Ein-Tim) [inline src/data/faq.json:44]:
```suggestion
                            "The TAN hotline will help you to enter the TAN if you have a positive PCR test result and if you have not received a TAN or document with a PCR-Test QR code. The TAN hotline is available Monday to Sunday from 7 a.m. to 8 p.m. German local time (except on German national holidays)",
```

File: src/data/faq.json [hunk 2]
```diff
@@ -20,8 +20,8 @@
...
+                            "The technical hotline (<a href='#international_phone_numbers' target='_blank' >click here to see the phone numbers</a>) will help you with technical questions about the Corona-Warn-App, for instance if you have problems with exposure logging. You can reach the technical hotline from Monday to Saturday from 7 a.m. to 8 p.m. (except on German national holidays), and it is free of charge for you within Germany.",
+                            "The TAN hotline (<a href='#international_phone_numbers' target='_blank' >click here to see the phone numbers</a>) will help you to enter the TAN if you have a positive PCR test result and if you have not received a TAN or document with a PCR-Test QR code. The TAN hotline is available Monday to Sunday from 7 a.m. to 8 p.m. German local time (except on national holidays). The call is free of charge from within Germany.",
```
Comment (Ein-Tim) [inline src/data/faq.json:24]:
```suggestion
                            "The TAN hotline (<a href='#international_phone_numbers' target='_blank' >click here to see the phone numbers</a>) will help you to enter the TAN if you have a positive PCR test result and if you have not received a TAN or document with a PCR-Test QR code. The TAN hotline is available Monday to Sunday from 7 a.m. to 8 p.m. German local time (except on German national holidays). The call is free of charge from within Germany.",
```

**Review by Ein-Tim — CHANGES_REQUESTED** [review]
Add "German"

**Comment by MikeMcC399** [comment]
@browkin 

According to the Android app 2.16.0-RC6 the technical hotline is still reachable until 10 PM (22:00 Uhr).

See https://github.com/corona-warn-app/cwa-website/issues/2294#issuecomment-1007542853

---
Three-dot icon > App-Informationen > Technische Hotline

Technische Hotline

@dsarkar should confirm

File: src/data/faq.json (hunk 2 above)
Comment (browkin) [reply src/data/faq.json]: @Ein-Tim Thank you! Added.

**Comment by Ein-Tim** [comment]
@dsarkar Can you confirm that in the future, the TAN hotline will be available for a shorter amount of time than the technical hotline?

**Comment by dsarkar** [comment]
@Ein-Tim I forwarded the question. Stand by for answer.

**Comment by dsarkar** [comment]
Times are confirmed: No change regarding technical hotline, and TAN hotline will be the hour as shown above.

**Comment by Ein-Tim** [comment]
@dsarkar Thanks, okay.

**Comment by MikeMcC399** [comment]
@dsarkar 
Did you find out the date that the change to the TAN hotline opening hours is planned to take place? Is it on the day that 2.16 is released or some other day?

**Comment by dsarkar** [comment]
@Ein-Tim @MikeMcC399 Change will come into effect on 16.01.2022. I have further information about the decision process.

**Comment by MikeMcC399** [comment]
@dsarkar 
based on ...
> Times are confirmed: No change regarding technical hotline, and TAN hotline will be the hours as shown above.

and
>  Change will come into effect on 16.01.2022.

this PR could come out of draft status. It looks now to correspond to the above.

I guess you would leave it with the label "On hold" or "do not merge" until Jan 16, 2022?

**Comment by MikeMcC399** [comment]
- I suggest to change the title of this PR to include the phrase "TAN hotline" so it is clear what phone number is affected.

- I suggested this also in issue https://github.com/corona-warn-app/cwa-website/issues/2294

File: src/data/faq.json (hunk 1 above)
Comment (browkin) [reply src/data/faq.json]: @Ein-Tim Thank you! added.

**Comment by browkin** [comment]
continued in PR #2303 2303

**Comment by dsarkar** [comment]
Should be merged 16. Jan 2021, before release of R 2.16. therefore reopen this PR and closing PR #2303

**Review by dsarkar — APPROVED** [review]

**Pull request merged**